src/topology/constructions.lean

```diff
@@ -628,6 +628,11 @@ end sum
 section subtype
 variables [topological_space α] [topological_space β] [topological_space γ] {p : α → Prop}
 
+lemma inducing_coe {b : set β} : inducing (coe : b → β) := ⟨rfl⟩
+
+lemma inducing_of_inducing_cod_restrict {f : α → β} {b : set β} (hb : ∀ a, f a ∈ b)
```

Comment:
```suggestion
lemma inducing.of_cod_restrict {f : α → β} {b : set β} {hb : ∀ a, f a ∈ b}
```
In this way, you can use dot notation.

Reply:
Totally agree for the name but why the curly brackets? When can lean guess that?
```suggestion
lemma inducing.of_cod_restrict {f : α → β} {b : set β} (hb : ∀ a, f a ∈ b)
```

Reply:
I will start merging this to move on with the other PRs. If the parentheses are important feel free to stop the merge

Reply:
The parentheses are not really important. It would be better with the curly brackets, as I wrote, because `hb` can already be seen explicitly in the next argument `h` so there's no need to write it twice. In the way it's currently written, it will just lead to useless underscores when you apply the lemma. But since you have already sent it to bors, let's not stop the build for that.

Reply:
Oh, I see, I think I wrote it that way because the only place where I used that was with an `apply` and so the last argument was not passed and lean could not guess it on its own, but I will for sure change it as you say the next time I use it in the standard way!